src/Microsoft.PowerShell.Commands.Utility/commands/utility/MatchString.cs

```diff
@@ -972,7 +972,8 @@ void IContextTracker.TrackEOF()
         /// <summary>
         /// Gets or sets the current pipeline object.
         /// </summary>
-        [Parameter(ValueFromPipeline = true, Mandatory = true, ParameterSetName = "Object")]
+        [Parameter(ValueFromPipeline = true, Mandatory = true, ParameterSetName = "ObjectQuiet")]
```

Comment:
Why are these sets named `ObjectQuiet` and `FileQuiet`? Isn't the normal behaviour with these parameter sets to output the MatchInfo objects? 

It should only need to be named after the `-Quiet` switch if that switch is the defining mandatory parameter for that set -- in this case it's just an optional switch for any of these sets.

I'd recommend just using `Object` / `File` / `ObjectRaw` / `FileRaw` names, as the `Quiet` appears to imply the default behaviour is `-Quiet`, which would probably not be desirable for future maintainability. 🙂

Reply:
I'll have to look into this one later, thank you.

Reply:
Yes, absolutely. After a good night's sleep, this makes lots of sense now. I've removed the 'Quiet' part of the parameternames, as you suggested. Good catch.